src/Servers/Kestrel/samples/http2cat/Program.cs

```diff
@@ -18,6 +18,8 @@ public static async Task Main(string[] args)
         using var host = new HostBuilder()
             .ConfigureLogging(loggingBuilder =>
             {
+                loggingBuilder.AddFilter((category, level) =>
+                    level >= LogLevel.Information || category.StartsWith("Microsoft.AspNetCore.Http2Cat", StringComparison.Ordinal));
```

Comment:
I thought the default was info?

Reply:
Empirically, not including that seems to drop everything that doesn't match "Http2Cat".

Reply:
I see Http2Cat logs locally when running http2cat sample.

> info: Microsoft.AspNetCore.Http2Cat.Http2CatHostedService[0]
      Connected to '[::1]:5001'.
info: Microsoft.AspNetCore.Http2Cat.Http2CatHostedService[0]
      Starting TLS handshake.
info: Microsoft.AspNetCore.Http2Cat.Http2CatHostedService[0]
      TLS handshake completed successfully.
info: Microsoft.AspNetCore.Http2Cat.Http2CatHostedService[0]
      Initialized http2 connection. Starting stream 1.

Reply:
It was probably just missing because of the exception I was seeing...